tests/sentry/tasks/test_backfill_seer_grouping_records.py

```diff
@@ -1724,3 +1725,26 @@ def test_backfill_seer_grouping_records_manually_skip_project(self, mock_logger)
             call("backfill finished, no cohort", extra={"project_id": self.project.id}),
         ]
         assert mock_logger.info.call_args_list == expected_call_args_list
+
+    @with_feature("projects:similarity-embeddings-backfill")
+    @patch("sentry.tasks.embeddings_grouping.utils.logger")
+    @patch("sentry.tasks.embeddings_grouping.utils._make_postgres_call")
+    def test_backfill_seer_grouping_records_postgres_exception(
+        self, mock_make_postgres_call, mock_logger
+    ):
+        """
+        Test log after postgres query retries with decreased batch size
+        """
+        mock_make_postgres_call.side_effect = OperationalError
```

Comment:
Tiny nit: Do you know that you can set this straight in the `@patch` decorator? (If you're choosing not to, that's fine (and I'd be curious to hear why), but have seen this pattern in a few tests and keep meaning to bring it up.)

Reply:
no reason! I'll change this and follow up to change the others as well